AWS-Certified-Solutions-Architect-Associate-Domain1.md

```
@@ -0,0 +1,714 @@
+# AWS Certified Solutions Architect - Associate (SAA-C03) 完全ガイド
```

markdownlint-cli2 (0.23.0)

[warning] Headings should be surrounded by blank lines
Expected: 1; Actual: 0; Below

(MD022, blanks-around-headings)